scss/_popover.scss

```diff
@@ -21,69 +21,66 @@
 
   // Popover directions
 
-  &.popover-top,
-  &.bs-tether-element-attached-bottom {
-    margin-top: -$popover-arrow-width;
+  &.bs-popover-top {
+    margin-bottom: $popover-arrow-width;
 
-    &::before,
-    &::after {
+    ::before,
```

Comment:
this is inconsistent. In this file sometimes we use `.arrow` and sometimes pseudo-elements to style "arrows"